index.html

```diff
@@ -2121,9 +2121,9 @@ <h3 id="Schema-Building">Schema Building:</h3>
 
 
     <p id="Schema-dropPrimary">
-      <b class="header">dropPrimary</b><code>table.dropPrimary()</code>
+      <b class="header">dropPrimary</b><code>table.dropPrimary(constraintName)</code>
```

Comment:
Since `constraintName` is optional, please format like this:

```
table.dropPrimary([constraintName])
```

or

```
table.dropPrimary([constraintName="tablename_pkey"])
```

Your preference.